packages/babel-types/src/definitions/core.js

```diff
@@ -1401,6 +1401,12 @@ defineType("ExportAllDeclaration", {
     source: {
       validate: assertNodeType("StringLiteral"),
     },
+    assertions: {
```

Comment:
`assertions` should be optional, otherwise it is a breaking change.

Also we need to update `packages/babel-parser/ast/spec.md`